.github/workflows/spack_ci_bridge.yml

```
@@ -1,6 +1,10 @@
 name: spack_ci_bridge
 
-on: [push, pull_request]
+on:
+  pull_request:
+    tags-ignore:
+      - flux-sync
+      - flux-write-check-**
```

Ah, it looks like `tags-ignore` is only supported for `push` events, not `pull_request` https://docs.github.com/en/github-ae@latest/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore, which is why there's no CI run for the latest commit. 

I think changing to

```suggestion
on:
  push:
    tags-ignore:
      - flux-sync
      - flux-write-check-**
    branches-ignore:
      - main
      # Maybe flux and production here as well?
```

would have the same effect, right?

---

Good catch. Or maybe just do `on: pull_request` and remove the CI on push altogether?